### ISSUE TITLE
global bot notification settings shouldn't overwrite room notification settings

### ISSUE THREAD

**Issue by schiessle** [vector-im/element-meta#1576]

**Comment by downeymj** [comment]
I agree. I would love to take advantage of notifications from certain bots like Go-NEB (RSS) etc., but right now with the "one size fits all" bot notification setting, I can only get notifications from all bots in all rooms that I'm in, or no notifications at all. So that means I have to disable them all, otherwise all the GitHub bots that 80% of projects have in their rooms notify me 24x7 too. 

Bot notifications should be a per-room toggle, not global -- and not only here in element-web but also in Android, etc.

**Comment by daniellekirkwood** [comment]
@giomfo @amshakal @VolkerJunginger 

An example of the confusion between "global" and "room" and where the override should sit.
